### PR TITLE
feat: wrap page

### PR DESCRIPTION
**Pull request by d0c0q**
## TODO
- [x] change image for wrap dialog
- [x] change address of pearl contract

### CODE REVIEW

File: src/store/slices/account-slice.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { JsonRpcProvider } from '@ethersproject/providers';
 import { createAsyncThunk, createSelector, createSlice } from '@reduxjs/toolkit';
 import { ethers } from 'ethers';
 import _ from 'lodash';
-import { ClamTokenContract, MAIContract, StakedClamContract } from '../../abi/';
+import { ClamTokenContract, MAIContract, StakedClamContract, PearlTokenContract } from '../../abi/';
```
Comment (poying) [inline src/store/slices/account-slice.ts]:
```suggestion
import { ClamTokenContract, MAIContract, StakedClamContract, PearlTokenContract } from 'src/abi';
```

File: src/store/slices/account-slice.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { JsonRpcProvider } from '@ethersproject/providers';
...
 import { BondKey, getAddresses, getBond } from '../../constants';
```
Comment (poying) [inline src/store/slices/account-slice.ts]:
```suggestion
import { BondKey, getAddresses, getBond } from 'src/constants';
```

File: src/store/slices/account-slice.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { JsonRpcProvider } from '@ethersproject/providers';
...
 import { contractForBond, contractForReserve, setAll } from '../../helpers';
```
Comment (poying) [inline src/store/slices/account-slice.ts]:
```suggestion
import { contractForBond, contractForReserve, setAll } from 'src/helpers';
```

File: src/views/Wrap/index.tsx [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+  const onApproval = async () => {
+    await dispatch(approveWrapping({ address, provider, networkID: chainID }));
+  };
+
+  const onChangeWrap = async (action: string) => {
+    if (isNaN(Number(quantity)) || Number(quantity) === 0 || quantity === '0') {
+      alert('Please enter a value!');
+    } else {
+      setAction(action);
+      let wrapTx: any = await dispatch(
```
Comment (poying) [inline src/views/Wrap/index.tsx:120]: Try not to use `any`

File: src/store/slices/account-slice.ts (hunk 1 above)
Comment (d0c0q) [reply src/store/slices/account-slice.ts]: https://github.com/OtterClam/otter-frontend/pull/27/commits/cb4a331e45581b71432a32b6ff71ccca3ee81c75

File: src/store/slices/account-slice.ts (hunk 2 above)
Comment (d0c0q) [reply src/store/slices/account-slice.ts]: https://github.com/OtterClam/otter-frontend/pull/27/commits/cb4a331e45581b71432a32b6ff71ccca3ee81c75

File: src/store/slices/account-slice.ts (hunk 3 above)
Comment (d0c0q) [reply src/store/slices/account-slice.ts]: https://github.com/OtterClam/otter-frontend/pull/27/commits/cb4a331e45581b71432a32b6ff71ccca3ee81c75

File: src/views/Wrap/index.tsx (hunk 4 above)
Comment (d0c0q) [reply src/views/Wrap/index.tsx:120]: https://github.com/OtterClam/otter-frontend/pull/27/commits/864af78ceb9dbcc8f7239ebbd56095d657f90a3f

Comment (d0c0q) [reply src/views/Wrap/index.tsx:120]: reference: https://redux-toolkit.js.org/tutorials/typescript

**Pull request merged**